database/factories/CatalogEntityFactory.php

```diff
@@ -6,15 +6,14 @@
 use Faker\Provider\en_US\PhoneNumber;
 use Faker\Provider\en_US\Company;
 
+use App\User;
```

Comment:
This should be derived from the repository, as we've done in other, recent instances. I'll fix it.